sdk/storage/Azure.Storage.Blobs/src/BlockBlobClient.cs

```diff
@@ -2160,6 +2160,195 @@ protected sealed override BlobBaseClient WithSnapshotCore(string snapshot)
         }
         #endregion OpenWrite
 
+        #region UploadFromUri
+        /// <summary>
+        /// The Put Blob from URL operation creates a new Block Blob where the contents of the
```

Comment:
Should this be updated to "The Upload From URL operation", or should we stick with what the REST API operation is called?

Reply:
Fixed.